Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.16-buster as builder
+RUN mkdir /go/upctl
```

Comment:
`WORKDIR` should create the directory too

Reply:
Cheers! Should have remembered but nope. :) Fixed now.